app/controllers/admin/claims_controller.rb

```diff
@@ -7,4 +9,10 @@ def index
       format.csv { send_file csv.file, type: "text/csv", filename: "claims.csv" }
     end
   end
+
+  private
+
+  def ensure_service_operator
+    redirect_to admin_auth_failure_path unless service_operator_signed_in?
```

Comment:
```suggestion
    render "admin/auth/failure", status: :unauthorized unless service_operator_signed_in?
```

So that the response code is correct?